Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -666,6 +665,44 @@ sourceUnserviceableConfig state =
   in
     errorModalConfig'
 
+waitTimeInfoCardConfig :: ST.HomeScreenState -> RequestInfoCard.Config
+waitTimeInfoCardConfig state = let
+  config = RequestInfoCard.config
+  requestInfoCardConfig' = config{
+    title {
+      text = if state.data.currentSearchResultType == ST.QUOTES then getString OTP_EXPIRE_TIMER else getString WAIT_TIMER,
```

Comment:
make `state.data.currentSearchResultType == ST.QUOTES` as variable and reuse it across the function. Also good to create a `either` function which will check for the result type and return the kind of string required automatically